packages/pages/src/vite-plugin/build/build.ts

```diff
@@ -62,11 +63,12 @@ const discoverInputs = async (
       const parsedPath = parse(template);
 
       if (parsedPath.ext === ".tsx" || parsedPath.ext === ".jsx") {
-        input[`hydrate/${parsedPath.name}`] =
-          `${hydrationOutputDir}/${template}`.replace("jsx", "tsx");
+        input[`hydrate/${parsedPath.name}`] = path
```

Comment:
Do the keys need to be transformed?

Reply:
the longer answer is that rollup handles these correctly already.